tests/manage/pv_services/test_bz1729853.py

```diff
@@ -0,0 +1,198 @@
+# -*- coding: utf8 -*-
```

Comment:
Can you please give this python script a better test name,  test_bzxxxxx.py is not readable unless one goes and digs it.

Reply:
Ok, will do.